backend/dataall/modules/dataset_sharing/db/share_object_repositories.py

```diff
@@ -562,6 +562,33 @@ def list_user_received_share_requests(session, username, groups, data=None):
                 )
             )
         )
+        # TODO: search functionality
```

Comment:
Clean up TODO if we do not plan to add search functionality for share views and just keep to filters